packages/web-components/src/components/ic-link/ic-link.css

```diff
@@ -33,7 +33,7 @@
   text-decoration: none;
 }
 
-@supports (text-decoration-thickness: 25%) {
+@supports (text-underline-offset: 25%) {
```

Comment:
Does the `text-underine-offset` property on line 43 need to be 25% now?

Reply:
Good spot, I just copied it over, this line should be 10%